packages/connect/src/filter.ts

```diff
@@ -207,6 +207,7 @@ export class Filter {
       // If private address connections are explicitly allowed, do not dial them
       if (!this.opts.allowPrivateConnections) {
         // Do not pollute logs by rejecting private address connections attempts
+        console.log(`private connection ${ma.toString()}`)
```

Comment:
Should be using `debug` or log facility.

Reply:
removed the line since it is a leftover

Reply:
changed